src/libraries/System.Runtime.Extensions/tests/System/EnvironmentTests.cs

```diff
@@ -181,7 +181,7 @@ public void OSVersion_ParseVersion(string input, int major, int minor, int build
             Assert.Equal(expected, actual);
         }
 
-        [Fact]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotMacOsAppleSilicon)] // [ActiveIssue("https://github.com/dotnet/runtime/issues/49106")]
```

Comment:
Active issue can take a condition as well, please use `ActiveIssue` instead of adding a comment with the `ActiveIssueAttribute`. 

https://github.com/dotnet/arcade/blob/26b0c6764841fd06d8e7322ccd8581652ba22870/src/Microsoft.DotNet.XUnitExtensions/src/Attributes/ActiveIssueAttribute.cs#L23

It would be like `[ActiveIssue("url", typeof(PlatformDetection), nameof(PlatformDetection.IsNotMacOsAppleSilicon)]`

Reply:
Strange grepping the existing `ActiveIssue` attributes, I didn't see that pattern.  This is the typical pattern I saw....

I'll change this one.